config/database.ini

```diff
@@ -11,3 +11,8 @@ INCLUDE = ./config/database/sqlserver.ini
 # File containing EOSERV's database schema
 # Use ./install.sql for MySQL/SQLite
 InstallSQL = ./install_sqlserver.sql
+
+## AutoCreateDatabase
+# True to automatically try to create the database if it doesn't exist (MySQL and SqlServer only)
+#
```

Comment:
Is this line consisting only of a # character intentional? It seems to deviate from the conventions established by prior lines/sections.